.github/workflows/github-release.yml

```diff
@@ -16,23 +16,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Python 3.9
+      - name: Set up Python 3.10
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9
+          python-version: 3.10
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           pip install .
-      - name: "Build & test"
-        run: |
-          echo "done!"
-
-      - uses: "marvinpinto/action-automatic-releases@latest"
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Create Release
+        uses: "marvinpinto/action-automatic-releases@latest"
         with:
           repo_token: "${{ secrets.GITHUB_TOKEN }}"
           automatic_release_tag: "latest"
           prerelease: false
           files: |
             LICENSE.txt
```

Comment:
So, it is only LICENSE.txt that is included in the release now.

Reply:
I tested it just now. The resulting release can be found here: https://github.com/ZhengLin-Li/gcp_scanner/releases. 

It turned out that all source code is included. What else do we need to include then?

Reply:
@mshudrak hi, the below screenshot shows the release package.
![image](https://user-images.githubusercontent.com/63448884/222937686-909d7cef-5447-44b5-aee6-d439683bf33f.png)

is that all we need to include?

Reply:
This looks good but I am not sure about this third-party dependency: `marvinpinto/action-automatic-releases@latest`. Is there any Github native mechanism in order to support release process?

Reply:
It seams no. I have searched GitHub native mechanism and did not found one.

Reply:
I think that introducing additional reliance on third-party project is not a good idea for that task from security and reliability point of view.

Reply:
I found that GitHub has API that can create a release: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release

I will investigate on this and check if it works for our case.